src/App.js

```diff
@@ -35,36 +39,52 @@ function App() {
             <main>
                 <Bag
                     bagLabel="best seller"
-                    imgbagImage={bag1}
+                    bagImage={bag1}
                     bagTitle="bag1"
                     bagName="The handy bag"
                     bagPrice="€400,-"
```

Comment:
In de aandachstpuntenlijst zul je vast de opmerking betreffende de prijs hebben gezien:
<h4>€{price},-</h4> zou dan de gewenste oplossing zijn.